### PR TITLE
All Contributors cleanup

### PR DESCRIPTION
**Pull request by bourque**
This PR updates the all-contributors part of the `README` to remove individual contribution items. 

I think specifying individual contribution types is probably a bit overkill and requires too much oversight. We all contribute to this project in many intangible ways anyways, so I would argue just a straight list of contributors makes more sense.

### CODE REVIEW

**Review by sdhoyt — APPROVED** [review]
I agree. Thanks for updating this!

**Review by tech3371 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,21 +32,21 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/vmartinez-cu"><img src="https://avatars.githubusercontent.com/u/39746325?v=4?s=100" width="100px;" alt="vmartinez-cu"/><br /><sub><b>vmartinez-cu</b></sub></a><br /></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/subagonsouth"><img src="https://avatars.githubusercontent.com/u/16110870?v=4?s=100" width="100px;" alt="Tim Plummer"/><br /><sub><b>Tim Plummer</b></sub></a><br /></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/daralynnrhode"><img src="https://avatars.githubusercontent.com/u/143308810?v=4?s=100" width="100px;" alt="Daralynn Rhode"/><br /><sub><b>Daralynn Rhode</b></sub></a><br /></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/anamanica"><img src="https://avatars.githubusercontent.com/u/171708990?v=4?s=100" width="100px;" alt="anamanica"/><br /><sub><b>anamanica</b></sub></a><br /></td>
```
Comment (tech3371) [inline README.md]: Can we spell out Ana's name too?

File: README.md [hunk 2]
```diff
@@ -32,21 +32,21 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/tech3371"><img src="https://avatars.githubusercontent.com/u/36522642?v=4?s=100" width="100px;" alt="Tenzin Choedon"/><br /><sub><b>Tenzin Choedon</b></sub></a><br /></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/laspsandoval"><img src="https://avatars.githubusercontent.com/u/46567335?v=4?s=100" width="100px;" alt="Laura Sandoval"/><br /><sub><b>Laura Sandoval</b></sub></a><br /></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/sdhoyt"><img src="https://avatars.githubusercontent.com/u/7146374?v=4?s=100" width="100px;" alt="Sean Hoyt"/><br /><sub><b>Sean Hoyt</b></sub></a><br /></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/GFMoraga"><img src="https://avatars.githubusercontent.com/u/104743000?v=4?s=100" width="100px;" alt="Gabriel M."/><br /><sub><b>Gabriel M.</b></sub></a><br /></td>
```
Comment (tech3371) [inline README.md]: same for Gabriel and others?

**Review by tech3371 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bourque) [reply README.md]: Good call!

**Review by tech3371 — APPROVED** [review]
LGTM! Thank you for doing this!

**Pull request merged**